### PR TITLE
[multiple] Fix unnecessary_import lint in repo.

### PR DESCRIPTION
**Pull request by ditman**
Several packages are failing CI in `master` due to a new lint: `unnecessary_import`:

https://github.com/flutter/packages/runs/3681767672

This PR fixes the affected packages, so they pass the new lint and CI can continue running normally.

Part of: https://github.com/flutter/flutter/issues/74381

## Pre-launch Checklist

- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated `CHANGELOG.md` to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the `#hackers-new` channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by ditman** [comment]
(@stuartmorgan note that there's a small "quirk" with the `version_check` command [here](https://github.com/flutter/plugins/blob/79595de6752d75e226d601e275776e709e343f69/script/tool/lib/src/version_check_command.dart#L354), that made me do [this change](https://github.com/flutter/packages/pull/473/commits/60a41ecef0ee0d01ed3c2ebbeefbf1c8000f7951).

Not sure if this is a bug or a feature (to me, it's a feature requiring `##`, but the tool crashed when parsing the changelog)

File: packages/metrics_center/CHANGELOG.md [hunk 1]
```diff
@@ -1,41 +1,47 @@
-# 1.0.1
+## NEXT
+
+- Fix `unnecessary_import` lint errors.
+- Update version titles in CHANGELOG.md so plugins tooling understands them.
```
Comment (stuartmorgan) [inline packages/metrics_center/CHANGELOG.md:4]: I'm not sure CHANGELOG needs to describe CHANGELOG changes; that's a bit meta :)

**Review by stuartmorgan — APPROVED** [review]
LGTM, thanks! I should have remembered flutter/packages when I reviewed the flutter/plugins change.

**Comment by stuartmorgan** [comment]
> Requiring the `##` is a good feature, the only problem is that the version check command crashed when parsing the rest of the CHANGELOG file. [Error log](https://cirrus-ci.com/task/6701603815161856?logs=version#L12-L26).

Yeah, the crash is a bug. I'll fix that, thanks!

**Pull request merged**